.gitignore

```diff
@@ -7,4 +7,5 @@ node_modules/
 ormconfig.json
 ormlogs.log
 npm-debug.log
+test/functional/sqljs/*.sqlite
```

Comment:
Why not use temp folder? It has it own gitignore file.

Reply:
That output had previously gone to `build/compiled/test...` which is gitignored already. I could redirect this output, but that would involve changing more tests. Think that would be better?

Reply:
If that how it always worked let's leave it for now.